### DISCUSSION TITLE
NBA Stadiums

### DISCUSSION THREAD

**Discussion by meleakewubbie** [jakobzhao/geog495#78] [Final Project]
Application: https://meleakewubbie.github.io/geog495_final.github.io/index.html 
Github: https://github.com/meleakewubbie/geog495_final.github.io

My application displays NBA stadiums. Clicking on a marker will give the stadium name, city, team conference, and team division. 

<img width="1280" alt="screenshot" src="https://user-images.githubusercontent.com/77419851/146429272-06cae323-b02f-4348-85a8-e0631fc8a384.png">
<img width="753" alt="screenshot2" src="https://user-images.githubusercontent.com/77419851/146429291-68611f85-9832-419e-9004-cdfa027d0903.png">

**Comment by ChristopherBanas** [comment]
really cool! never realized how far out Portlands stadium was in comparison to the rest of the nba
